.github/workflows/integration-tests-against-emulator.yaml

```diff
@@ -0,0 +1,19 @@
+on:
```

Comment:
If you can update this to reflect the new style in https://github.com/googleapis/java-spanner/blob/master/.github/workflows/integration-tests-against-emulator.yaml, that would be awesome.

Reply:
Done.